app/assets/stylesheets/base.scss

```diff
@@ -11,26 +11,36 @@ a {
 
   &:focus,
   &:hover {
-    border-bottom: 1px dotted $sul-link-color-border;
+    border-bottom: 1px dotted $sul-link-border-color;
     color: $black;
     text-decoration: none;
   }
 }
 
 body {
+  background-color: $sul-body-bg-color;
+  color: $black;
   font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
+  font-size: $font-size-base;
+  line-height: $body-line-height;
+}
+
+h1,
+h2 {
+  font-weight: 300;
 }
 
 h1 {
   color: $sul-h1-font-color;
-  font-weight: 300;
+  font-size: 2.3em;
   letter-spacing: -0.05em;
   padding: 10px 0;
 }
 
 h2 {
   border-bottom: 1px solid $sul-h2-border-color;
-  color: $cardinal-red;
+  color: $sul-h2-font-color;
+  font-size: 1.6em;
   letter-spacing: 0;
   line-height: 1.2em;
   margin: 0.5em 0;
```

Comment:
Correct, the underline should only be on the home page. 

It also looks like the name above the button is using the global h2 margin, when it should be  margin: 0 0 1em 0; - 4th bullet in #93.